AUTHORS

```diff
@@ -109,3 +109,4 @@ Jacob Greenleaf <jacob@jacobgreenleaf.com>
 Alex Lourie <alex@instaclustr.com>; <djay.il@gmail.com>
 Marco Cadetg <cadetg@gmail.com>
 Martin Sucha <martin.sucha@kiwi.com>; <git@mm.ms47.eu>
+Ivan Boyarkin <ivan.boyarkin@kiwi.com>; <mr.vanboy@gmail.com>
```

Comment:
Not sure if I should be mentioned here 🤷‍♂ , but `CONTRIBUTING.md` contains the next line:
```
Your name and email have been added to the AUTHORS file (for copyright purposes)
```